core/tests/karma.conf.js

```diff
@@ -12,10 +12,10 @@ module.exports = function(config) {
     // jasmine-jquery is used to load contents of external JSON files in tests.
     frameworks: ['jasmine-jquery', 'jasmine'],
     files: [
-      'core/tests/karma-globals.js',
+      'local_compiled_js/core/tests/karma-globals.js',
```

Comment:
Does this work when you run with --prod_env?

Reply:
I have added a compile option in build which compiles ts files to `local_compiled_js` whenever frontend or e2e test are run. This ensures that local_compiled_js is present irrespective of prod_mode when we are testing. So, yes it works in prod mode also.